virtool/data/events.py

```diff
@@ -51,7 +52,16 @@ class _InternalEventsTarget:
     q = asyncio.Queue(maxsize=1000)
 
     def emit(self, event: Event):
-        self.q.put_nowait(event)
+        retries = 3
+        for attempt in range(retries-1):
```

Comment:
* Unnecessary variable `retries`.
* Unused variable `attempt`.
* Current code only tries two times, whereas I believe we discussed three tries.

```suggestion
        for _ in range(3):
```